android/src/main/java/com/rnmapbox/rnmbx/utils/ViewTagResolver.kt

```diff
@@ -32,7 +32,7 @@ open class ViewTagResolver(val context: ReactApplicationContext) {
                 try {
                     val view = manager.resolveView(viewTag)
 
-                    list.forEach { it.fn(view) }
+                    list.forEach { view?.let { itView -> it.fn(view) } }
```

Comment:
[nit] since `itView` is unused, we can replace it with `_`. better approach is probably just to filter out nulls instead:

```suggestion
                    list
                        .filterNotNull()
                        .forEach { it.fn(view) }
```

Reply:
Actually, that's a typo, it should be:

```suggestion
                    list.forEach { view?.let { itView -> it.fn(itView) } }
```

This fixed a build failure when we upgraded to Expo 52